com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -114,42 +123,112 @@ public PostProcessPass(RenderPassEvent evt, PostProcessData data, Material blitM
             else // Expect CoC banding
                 m_GaussianCoCFormat = GraphicsFormat.R8_UNorm;
 
-            // Bloom pyramid shader ids - can't use a simple stackalloc in the bloom function as we
-            // unfortunately need to allocate strings
-            ShaderConstants._BloomMipUp = new int[k_MaxPyramidSize];
-            ShaderConstants._BloomMipDown = new int[k_MaxPyramidSize];
-
-            for (int i = 0; i < k_MaxPyramidSize; i++)
-            {
-                ShaderConstants._BloomMipUp[i] = Shader.PropertyToID("_BloomMipUp" + i);
-                ShaderConstants._BloomMipDown[i] = Shader.PropertyToID("_BloomMipDown" + i);
-            }
-
             m_MRT2 = new RenderTargetIdentifier[2];
             m_ResetHistory = true;
             base.useNativeRenderPass = false;
+
+            m_TempTarget = RTHandles.Alloc(
```

Comment:
You should allocate texture upon usage, otherwise you keep resources allocated even though not used.